lms/djangoapps/grades/rest_api/v1/gradebook_views.py

```diff
@@ -332,14 +333,14 @@ class GradebookView(GradeViewMixin, PaginatedAPIView):
     **Example Request**
         GET /api/grades/v1/gradebook/{course_id}/                       - Get gradebook entries for all users in course
         GET /api/grades/v1/gradebook/{course_id}/?username={username}   - Get grades for specific user in course
-        GET /api/grades/v1/gradebook/{course_id}/?username_contains={username_contains}
+        GET /api/grades/v1/gradebook/{course_id}/?user_contains={user_contains}
```

Comment:
How strict are we about changing parameter names without changing the "API version"? (I'm guessing not very strict)

Reply:
Please do not break a released REST API like this. If we want to change the parameter because we think the naming is clearer, then please accept both parameters. But don't break existing clients.